Dockerfile

```diff
@@ -7,17 +7,23 @@ ENV FAH_VERSION_MAJOR=7.5
 ENV DEBIAN_FRONTEND=noninteractive
 
 RUN useradd -ms /bin/bash folder
+RUN mkdir -p /etc/OpenCL/vendors && \
+    echo "libnvidia-opencl.so.1" > /etc/OpenCL/vendors/nvidia.icd
 
-RUN apt-get update && apt-get install --no-install-recommends -y \
+RUN apt-get update && apt-get install --no-install-recommends -y nvidia-opencl-dev ocl-icd-opencl-dev clinfo rsync wget vim file less \
```

Comment:
Why did you include `rsync wget vim file less`?

Reply:
During Debugging I can remove.